packages/pwa-kit-dev/jest.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
     ...base,
     coverageThreshold: {
         global: {
-            branches: 67,
+            branches: 65,
```

Comment:
I had to lower this threshold since the new conditional in `build-dev-server.js` for setting the HMR publicPath dropped coverage below 67%. There is no easy way to test the new conditional since the changes are inside `_addSDKInternalHandlers` and our tests override `_addSDKInternalHandlers` with a no-op.

Reply:
fixed, bumped up to 68